package part2.package2;

import java.io.IOException;

public class App1 {
    public static void main(String[] args) throws IOException {
     //   throw new Exception(); // тут ошибка компиляции
    }
}
